webapp/src/ts/modules/analytics/analytics-targets.component.html

```diff
@@ -25,17 +36,6 @@ <h2 *ngIf="!target.translation_key">{{ target.name | translateFrom }}</h2>
           <p *ngIf="target.subtitle_translation_key">{{ target.subtitle_translation_key | translate }}</p>
         </div>
       </div>
-
-      <div class="body">
-        <mm-analytics-targets-progress *ngIf="target.type === 'percent'" [target]="target" [value]="target.value" [aggregate]="false"></mm-analytics-targets-progress>
-        <div class="count" *ngIf="target.type !== 'percent'">
-          <div class="goal" *ngIf="target.goal >= 0">
-            <label>{{ 'analytics.target.monthly_goal' | translate }}</label>
-            <p>{{ target.goal | localizeNumber }}</p>
-          </div>
```

Comment:
I made a change to the structure of this `div` based on the new design, but I need guidance on whether using one p tag is okay. @ngaruko

Reply:
@Benmuiruri . I think @latin-panda would be the best person to help you with that, but I can have a look at the new UI vs the existing UI and make a suggestion if something doesn't look properly aligned or something, if that would help?
On a different topic (e2e tests which is where I probably can make more of meaningful contribution), did you have a look at [this conversation](https://github.com/medic/cht-core/issues/8513#issuecomment-1732911722), we might need an addition test to check the color coding for the state of the target/goal (reached or not). - and re-organise these analytics tests.

Reply:
Thanks @ngaruko. I got clarification from Nicole about how to translate the Figma design to code. I will work on the design changes then when I get to testing I will add the test for the color code and re-organize the tests based on the conversation.